### PR TITLE
Empty file upload

### PR DESCRIPTION
**Pull request by yenienserrano**
**Description:**

When trying to import an empty file in rules or decoders, there was no information, it let you upload it but it had no effect, I added a warning to let the user know that he is uploading an empty file and not to let him upload it.

**Test:**

1. Navigate to `Management/rules or decoders`
2. Try uploading an empty file
3. See the warning

**Issue:**

- #4059 

**Screenshot:**

image

### CODE REVIEW

**Comment by Mayons95** [comment]
Test OK

Decoders:
https://user-images.githubusercontent.com/104914131/170778687-bc88cb86-f1d9-41e5-b6cb-56c01c892608.mp4

Rules:
https://user-images.githubusercontent.com/104914131/170778906-cc28f8e0-b1a2-4835-8f38-c17140c61530.mp4

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 ### Fixed
 
 - Fixed Wazuh Dashboard troubleshooting url [#4150](https://github.com/wazuh/wazuh-kibana-app/pull/4150)
+- Fixed import of an empty rule or decoder file [#4176](https://github.com/wazuh/wazuh-kibana-app/pull/4176)
```
Comment (AlexRuiz7) [inline CHANGELOG.md:10]: Must be changed to the 4.3.4 section.

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

**Review by AlexRuiz7 — APPROVED** [review]
LGTM!

**Review by asteriscos — APPROVED** [review]
Nitpick: I would change the message to something kinder, like "Please remove uploaded empty files to continue"
CR: :heavy_check_mark: 
Test: :heavy_check_mark: 
Screenshot from 2022-06-02 12-42-31

**Pull request merged**